### PR TITLE
Minor edits.

### PR DESCRIPTION
**Pull request by pauliesnug**
## Description
* Minor edits
* Changed naming scheme
* Added MarkdownLint

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: MarkdownLint
+        uses: github/super-linter@v4
+        env:
+          VALIDATE_ALL_CODEBASE: true
+          DEFAULT_BRANCH: main
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (pawlean) [inline .github/workflows/lint.yml]: Why is this needed?

**Review by pawlean — CHANGES_REQUESTED** [review]

File: .github/workflows/lint.yml (hunk 1 above)
Comment (pauliesnug) [reply .github/workflows/lint.yml]: Oops! Didn’t see that in the default config.

**Review by pawlean — COMMENTED** [review]
LGTM
